yarn.lock

```diff
@@ -27941,6 +27949,11 @@ schema-utils@^3.0.0:
     ajv "^6.12.5"
     ajv-keywords "^3.5.2"
 
+scrypt-js@2.0.4:
```

Comment:
Not sure what requires this particular version of this package, but ideally it would match the version already-in-use

Reply:
Fixed.